docs/user/alerting/alerting-getting-started.asciidoc

```diff
@@ -53,7 +53,7 @@ to control the details of the conditions to detect.
 
```

Comment:
I think one more change is still needed. Rules section, https://www.elastic.co/guide/en/kibana/7.16/alerting-getting-started.html#_rules, still mentions "domain rules". Maybe this use also needs to be updated to better reflect UI?